### PR TITLE
detatch tvm

### PR DESCRIPTION
**Pull request by qiuxiafei**

### CODE REVIEW

File: tao/tao_bridge/passes/tao_defuse_pass.cc [hunk 1]
```diff
@@ -1,291 +0,0 @@
-// Copyright 2021 The BladeDISC Authors. All rights reserved.
```
Comment (wyzero) [inline tao/tao_bridge/passes/tao_defuse_pass.cc:1]: We still need this on CPU?

File: tao/tao_bridge/passes/tao_optimization_pass.cc [hunk 2]
```diff
@@ -113,20 +113,6 @@ Status TaoOptimizationPass::Run(const GraphOptimizationPassOptions& options) {
 
   bool mlir_whole_graph_compilation =
       GetTaoBridgeOptions()->experimental_enable_mlir_whole_graph_compilation;
-  auto flags = GetMarkForCompilationPassFlags();
-  if (flags->tf_xla_cpu_global_jit) {
```
Comment (wyzero) [inline tao/tao_bridge/passes/tao_optimization_pass.cc:117]: not delete this part, we still need this on cpu.

File: tao/tao_bridge/passes/tao_defuse_pass.cc (hunk 1 above)
Comment (qiuxiafei) [reply tao/tao_bridge/passes/tao_defuse_pass.cc:1]: This whole pass will be skipped if `use_tvm_` is `false` on L268.

Comment (qiuxiafei) [reply tao/tao_bridge/passes/tao_defuse_pass.cc:1]: Well,  i saw it's always `true`...

File: tao/tao_bridge/passes/tao_optimization_pass.cc (hunk 2 above)
Comment (qiuxiafei) [reply tao/tao_bridge/passes/tao_optimization_pass.cc:117]: Ok,  but `TaoCloneConstantsForBetterClusteringPass` will be deleted.

Comment (qiuxiafei) [reply tao/tao_bridge/passes/tao_optimization_pass.cc:117]: TaoCloneConstantsForBetterClusteringPass reserved.

**Review by wyzero — APPROVED** [review]
LGTM

**Review by wyzero — APPROVED** [review]

**Pull request merged**